DevEdu.API/DevEdu.DAL/Repositories/TagRepository.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Dapper;
+using DevEdu.DAL.Models;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DevEdu.DAL.Repositories
+{
+    public class TagRepository : BaseRepository, ITagRepository 
+    {
+        private const string _tagInsertProcedure = "[dbo].[Tag_Insert]";
+        private const string _tagDeleteProcedure = "[dbo].[Tag_Delete]";
+        private const string _tagSelectAllProcedure = "[dbo].[Tag_SelectAll]";
+        private const string _tagSelectByIDProcedure = "[dbo].[Tag_SelectByID]";
+        private const string _tagUpdateProcedure = "[dbo].[Tag_Update]";
+        public int AddTag(TagDto tagDto)
+        {
+            return _connection.QuerySingleOrDefault<int>(_tagInsertProcedure, new { tagDto.Name }, commandType: CommandType.StoredProcedure);
+        }
+        public void DeleteTag(int id)
+        {
+            _connection.Execute(_tagDeleteProcedure, new { id }, commandType: CommandType.StoredProcedure);
+        }
+        public List<TagDto> SelectAllTags()
+        {
+            List<TagDto> tagDtos = _connection.Query<TagDto>(_tagSelectAllProcedure, commandType: CommandType.StoredProcedure).ToList();

```

Comment:
please use var instead of List<TagDto> 
and please stick to this formatting: 
![image](https://user-images.githubusercontent.com/18329725/124524458-c8333780-de03-11eb-88c8-71c5d68a7ba5.png)